phoebusgen/widget/properties.py

```diff
@@ -559,8 +559,8 @@ def predefined_line_color(self, name: object) -> None:
 
         :param name: <phoebusgen.colors> Predefined color name
         """
-        if (self.root.attrib['type'] == 'group'):
-            self.version('3.0.0')
+        if (self.root.attrib['type'] == 'group' and int(self.root.attrib['version'].split('.')[0]) < 3):
+            print('Line color not compatible with group widget version less than 3.0.0.')
```

Comment:
we need to add a return statement here so the code below doesn't execute. Right now it prints the error but still adds the line color prop

```python
>>> import phoebusgen
>>> phoebusgen.phoebus_version
'4.7.3'
>>> phoebusgen.change_phoebus_version("4.7.2")
Phoebus version manually changed to 4.7.2.
>>> b = phoebusgen.widget.Group("test", 2,2,2,2)
>>> b
<?xml version="1.0" ?>
<widget type="group" version="2.0.0">
  <name>test</name>
  <x>2</x>
  <y>2</y>
  <width>2</width>
  <height>2</height>
</widget>
>>> b.predefined_line_color("OK")
Line color not compatible with group widget version less than 3.0.0.
>>> b
<?xml version="1.0" ?>
<widget type="group" version="2.0.0">
  <name>test</name>
  <x>2</x>
  <y>2</y>
  <width>2</width>
  <height>2</height>
  <line_color>
    <color name="OK" red="0" green="255" blue="0" alpha="255"/>
  </line_color>
</widget>
```

Also could you add a few test cases for this? maybe test with versions 2.0.0, 3.0.0, and like 3.1.2